Web UI/CognitiveSearch.UI/wwwroot/js/results.js

```diff
@@ -87,33 +86,110 @@ function AddMapPoints(results) {
     if (mapDataSource !== null) {
         // clear the data source, add new POIs and re-center the map
         mapDataSource.clear();
-        coordinates =  UpdatePOIs(results, mapDataSource);
+        coordinates = UpdatePOIs(results, mapDataSource);
         if (coordinates) {
-            resultsMap.setCamera ({ center: coordinates });
+            resultsMap.setCamera({ center: coordinates });
         }
     }
     else {
         //Create a data source to add it to the map 
-        mapDataSource = new atlas.source.DataSource();
+        mapDataSource = new atlas.source.DataSource(null, {
+            cluster: true,
+            clusterRadius: 45,
+            clusterMaxZoom: 15
+        });
         coordinates = UpdatePOIs(results, mapDataSource);
-        
+
         //Wait until the map resources are ready for first set up.
         resultsMap.events.add('ready', function () {
+            resultsMap.imageSprite.add('bubble-icon', '../images/single_bubble.png');
```

Comment:
this seems to be a relative path to the sprite.. would this work as expected in the prod app?

Reply:
Good point, I think it should since its in the wwwroot folder.  If there's output issues Ill update this, unless there's a better way of referencing this.  There's better ways in C#, but this seemed to work for the javascript call.